src/main/java/com/igeekinc/util/objectcache/ObjectManager.java

```diff
@@ -1,29 +1,27 @@
-/*
- * Copyright 2002-2014 iGeek, Inc.
- * All Rights Reserved
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- * http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
- 
+/*
+ * Copyright 2002-2014 iGeek, Inc.
+ * All Rights Reserved
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
 package com.igeekinc.util.objectcache;
 
+import com.igeekinc.util.ChangeModel;
+import com.igeekinc.util.CheckCorrectDispatchThread;

```

Comment:
It seems this line is duplicated import with line 27

Reply:
yes, it's duplicated, I've fixed it, thanks.